tests/Integration/Behaviour/Features/Scenario/ApiAccess/api_access_management.feature

```diff
@@ -0,0 +1,73 @@
+      | description  | a simple description |
+    Then api access "AA-1" should have the following properties:
+      | clientName   | Thomas               |
+      | apiClientId  | test-id              |
+      | enabled      | true                 |
+      | description  | a simple description |
+    When I edit api access "AA-1" with the following values:
+      | clientName   | Toto                 |
+      | apiClientId  | test-id-toto         |
+      | enabled      | false                |
+      | description  | another description  |
+    Then api access "AA-1" should have the following properties:
+      | clientName   | Toto                 |
+      | apiClientId  | test-id-toto         |
+      | enabled      | false                |
+      | description  | another description  |
+    When I edit api access "AA-1" with the following values:
+      | apiClientId  | test-id-toto-2       |
+    Then api access "AA-1" should have the following properties:
+      | apiClientId  | test-id-toto-2       |
+
+
+  Scenario: Create an api access with non unique properties:
+    When I create an api access "AA-2" with following properties:
+      | clientName   | Thomas2              |
+      | apiClientId  | test-id-2            |
+      | enabled      | true                 |
+      | description  | a simple description |
+    When I create an api access "AA-3" with following properties:
+      | clientName   | Thomas3              |
+      | apiClientId  | test-id-2            |
+      | enabled      | true                 |
+      | description  | a simple description |
+    Then I should get an error that clientId is not unique
+    When I create an api access "AA-4" with following properties:
+      | clientName   | Thomas2              |
+      | apiClientId  | test-id-3            |
+      | enabled      | true                 |
+      | description  | a simple description |
+    Then I should get an error that clientName is not unique
+
+  Scenario: Create an api access with blank properties:
+    When I create an api access "AA-1" with following properties:
+      | clientName   |                      |
+      | apiClientId  | test-id-1              |
+      | enabled      | true                 |
+      | description  | a simple description |
+    Then I should get an error that clientName is blank
+    When I create an api access "AA-2" with following properties:
+      | clientName   | Thomas-1               |
+      | apiClientId  |                      |
+      | enabled      | true                 |
+      | description  | a simple description |
+    Then I should get an error that apiClientId is blank
+    When I create an api access "AA-4" with following properties:
+      | clientName   | Thomas-5             |
+      | apiClientId  | test-id-5            |
+      | enabled      | true                 |
+      | description  |                      |
+    Then I should get an error that description is blank
```

Comment:
As I commented in the Context class NotBlank is not the only constraint checked, the max length also exists so the wording should be adapted and you should add some extra test cases with values too long so that the behat tests have exhaustive test cases